lib/scope.js

```diff
@@ -122,6 +122,14 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
                 scope.uses_eval = save_scope.uses_eval;
                 scope.directives = save_scope.directives;
             }
+            // issue 1753 - name mangling in for loop scope should consider parent scope
```

Comment:
Please mention: 
```
// Safari/Webkit bug workaround - loop init let variable shadowing argument.
```
For bonus points if you have the webkit bug URL, please add it here.